.github/copilot-instructions.md

```diff
@@ -0,0 +1,17 @@
+Use `make install` to compile the code
+Use `make lint` to run the linter
+Use `make unittest` to run all unit tests.
+The TEST parameter for unit tests and acceptance tests (e.g. `make unittest TEST=TestPrefix`) can be used to run test functions that match `^(TestAcc|TestUnit)TestPrefix`
+Only run acceptance tests with user consent, and always specify a test prefix when running `make acctest TEST=TestPrefix`
+When running acceptance tests you don't need to specify `TF_ACC=1` because `make acctest` already does that
+Always run `make` commands in `/workspaces/terraform-provider-power-platform` working directory
+Don't ever run `terraform init`
+Do not direcly edit the files under `/docs` because they are auto-generated from MarkdownDescription on schemas using `make userdocs`
+test files should have `_test` appended to their package name
+To run an example, `cd` to its working directory and run `terraform apply -auto-approve`
```

Comment:
```suggestion
To run an example, `cd` to its working directory and run `terraform apply -auto-approve`
Keep the file naming same like in other other place in the repository. 
Do not add comments in files with .tf and .go extensions unless ask to do so
```

Reply:
I don't agree with banning comments.  It just needs to write better comments.  
```suggestion
To run an example, `cd` to its working directory and run `terraform apply -auto-approve`
Comments on methods should provide more information about how to use it, its parameters, and expected results. Omit comments that don't substantially improve the readability of the code.
```